backend/.env.example

```
@@ -0,0 +1,19 @@
+# Copy this file to `.env` (git-ignored) and fill in real local values.
+# Never commit the real `.env` file.
+
+# Spring profile — "dev" is the default (see spring.profiles.active in
+# application.yml) and only needs setting explicitly for "prod".
+# SPRING_PROFILES_ACTIVE=dev
+
+# MySQL (local)
+DB_HOST=localhost
+DB_PORT=3306
+DB_NAME=school_management
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_NAME key should go before the DB_PORT key

(UnorderedKey)